app/src/pages/CardSubmission/WaitingForSubmissions.tsx

```diff
@@ -37,77 +37,78 @@ function WaitingForSubmissions() {
+                  {submittedSoFar}/{`${total} cards so far`}
+                </Typography>
+              </Grid>
+            </>
+          ),
+          [WaitingForSubmissionsState.SubmittedAssign]: (
             <Grid item>
-              {currentPlayer.role === PlayerRole.Host ? (
-                <Button
-                  size="large"
-                  variant="contained"
-                  color="primary"
-                  onClick={async () => {
-                    const players = teamsWithSequence(currentGame.players)
-                    await updateAllPlayers({
-                      variables: {
-                        gameId: currentGame.id,
-                        players: players.map(({ id, team, team_sequence }) => ({
-                          id,
-                          team,
-                          team_sequence,
-                        })),
-                      },
-                    })
-                    updateGameState({
-                      variables: {
-                        id: currentGame.id,
-                        state: GameStateEnum.TeamAssignment,
-                      },
-                    })
-                  }}
-                >
-                  Assign Teams
-                </Button>
-              ) : null}
+              All players submitted! As the host, you can now assign teams.
             </Grid>
-          )}
-        </>
-      ) : (
-        <>
-          <Grid item container justify="center">
-            Just waiting for everyone else...<div style={{ width: 4 }}></div>
-            {waitingForPlayers.map((player) => (
-              <>
-                <PlayerChip username={player.username || ""}></PlayerChip>
-                <div style={{ width: 4 }}></div>
-              </>
-            ))}
-          </Grid>
-          <Grid item>
-            <Typography variant="h5">
-              {submittedSoFar}/{`${total} cards so far`}
-            </Typography>
-          </Grid>
-          <Grid item>
-            <div style={{ marginTop: 50 }}>
-              <Fishbowl></Fishbowl>
-            </div>
+          ),
+          [WaitingForSubmissionsState.SubmittedWait]: (
+            <Grid item>
+              {`All players submitted, ${submittedSoFar} cards in total. Now we
+              are waiting on the host to start the game!`}
+            </Grid>
+          ),
+        }[state]
+      }
+      <Grid item container direction="column" spacing={2} alignItems="center">
+        {unscreenedCards.map((card, index) => (
+          <Grid item key={index}>
+            <ScreenCard card={card}></ScreenCard>
           </Grid>
-        </>
-      )}
+        ))}
+      </Grid>
+      {WaitingForSubmissionsState.SubmittedAssign === state ? (
+        <AssignTeamsButton />
+      ) : WaitingForSubmissionsState.Waiting === state &&
+        !unscreenedCards.length ? (
+        <div style={{ marginTop: 50 }}>
+          <Fishbowl />
+        </div>
```

Comment:
@namoscato i think you identified this in your other PR - but let's just always show the fish. would that make it simpler?

Reply:
Addressed in 76da83f5b121e2a4801ae40daf40146ab6808335.